### ISSUE TITLE
Chrome Unstable channel + this script = crash

### ISSUE THREAD

**Issue by rodrigograca31** [StylishThemes/GitHub-Dark-Script#64]
* **Browser**: Version 76.0.3788.1 (Official Build) dev (64-bit)
* **Operating System**: Linux Mint 18.3
* **Link to page with the issue**: https://github.com/chemerisuk/cordova-plugin-firebase-analytics
* **Screenshot**: 
![image](https://user-images.githubusercontent.com/1134310/57850769-b0893700-77d6-11e9-8e37-9545c8e3b5d3.png)

It only happens in some github pages. Like the one above....
My script is up to date and not that I'm using Chrome Unstable, that means it's crashing for me now and will crash for others whenever what is causing the issue goes to the Stable channel...

**Comment by silverwind** [comment]
Likely a issue with unstable Chrome itself. I often see weird behaviour in Chrome Canary which magically never makes it to the release channel.

**Comment by rodrigograca31** [comment]
> Likely a issue with unstable Chrome itself. I often see weird behaviour in Chrome Canary which magically never makes it to the release channel.

true, still I decided to mention it in case someone knew why or in case it reaches the stable channel....

**Comment by rodrigograca31** [comment]
P.S. I made a quick search in the Chromium project bug tracker first and didnt see anything obvious....
